proto/source_metadata.proto

```diff
@@ -79,6 +79,11 @@ message Filesystem {
   string email = 3;
 }
 
+message File {
+  string name = 1;
```

Comment:
**Question:** What is name in this context?

Reply:
file name